src/sentry/static/sentry/app/options.jsx

```diff
@@ -135,7 +135,7 @@ const definitions = [
   }
 ];
 
-const definitionsMap = _.indexBy(definitions, 'key');
+const definitionsMap = _.keyBy(definitions, def => def.key);
```

Comment:
keyBy accepts a string as the second arg, so you can do
` const definitionsMap = _.keyBy(definitions, 'key');`

https://lodash.com/docs/4.17.4#keyBy